perf.groovy

```diff
@@ -97,16 +97,16 @@ def osShortName = ['Windows 10': 'win10',
                         "${python} \"%WORKSPACE%\\artifacts\\Microsoft.BenchView.JSONFormat\\tools\\build.py\" git --branch %GIT_BRANCH_WITHOUT_ORIGIN% --type " + runType)
                         batchFile("${python} \"%WORKSPACE%\\artifacts\\Microsoft.BenchView.JSONFormat\\tools\\machinedata.py\"")
 
-                        batchFile("powershell -ExecutionPolicy ByPass -NoProfile eng/common/build.ps1 -restore -build -warnaserror:0 -ci -configuration Release /p:ConfigurationGroup=Release /p:BuildTests=true /p:BuildNative=false /p:Performance=true /m:1 /p:LogToBenchview=true /p:BenchviewRunType=${runType} /p:PerformanceType=Profile /p:PythonCommand=${python}")
-                        batchFile("powershell -ExecutionPolicy ByPass -NoProfile eng/common/build.ps1 -restore -build -warnaserror:0 -ci -configuration Release /p:ConfigurationGroup=Release /p:BuildTests=true /p:BuildNative=false /p:Performance=true /m:1 /p:LogToBenchview=true /p:BenchviewRunType=${runType} /p:PerformanceType=Diagnostic /p:PythonCommand=${python}")
+                        batchFile("build.cmd -performanceTest ${commonArgs} /p:LogToBenchview=true /p:BenchviewRunType=${runType} /p:PerformanceType=Profile /p:PythonCommand=${python}")
+                        batchFile("build.cmd -performanceTest ${commonArgs} /p:LogToBenchview=true /p:BenchviewRunType=${runType} /p:PerformanceType=Diagnostic /p:PythonCommand=${python}")
                     }
                 }
                 else {
                     def python = 'python3.5'
                     def benchViewName = isPR ? 'corefx private \$BenchviewCommitName' : 'corefx rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
                     steps {
                         //We need to specify the max cpu count to be one as we do not want to be executing performance tests in parallel
-                        shell("./build.sh --ci -release")
+                        shell("./build.sh --includetests ${commonArgs}")
```

Comment:
You're removing this argument above.

Reply:
Yeah the perf groovy script was just an example of my changes, will update that.